client/source/sass/main/project/_layout.scss

```diff
@@ -86,6 +86,7 @@
 
   .data {
     height: 30px;
+    margin-right: 10px;
     &.bordered {
```

Comment:
@jpuri i checked other page. Can please tell me on which page it is causing conflict?